### ISSUE TITLE
Animation broke VictoryStack

### ISSUE THREAD

**Issue by AlexGrump** [FormidableLabs/victory#1647]
When I set anomation for VictoryArea/VictoryBar and VictoryStack rendering with different data set breaks

https://codesandbox.io/s/serene-sun-ml76j?file=/src/App.js

**Comment by rsheldiii** [comment]
I'm having this issue as well using VictoryStack and two VictoryBar charts, which are trying to animate between two different data sets. If the data sets have the same number of bars on the X axis, it works fine, but otherwise it will not update.

**Comment by rsheldiii** [comment]
actually @AlexGrump if I remove the `animate={animate}` from the VictoryArea in your example the animations work again

**Comment by AlexGrump** [comment]
> actually @AlexGrump if I remove the `animate={animate}` from the VictoryArea in your example the animations work again

Yep. This is the behavior that I also discovered. Think isn't normal
